recipe/meta.yaml

```
@@ -22,7 +22,7 @@ requirements:
     - python  # [build_platform != target_platform]
   host:
     - cython >=0.29.13
-    - numpy
+    - numpy >=1.25
```

```suggestion
    - numpy
```

In principle let's try to use the global pinnings. If that doesn't work, we can try redefining them via `recipe/conda_build_config.yaml`, but that's usually more trouble than it's worth.